application/uitests/org.openjdk.jmc.flightrecorder.uitest/src/test/java/org/openjdk/jmc/flightrecorder/uitest/JfrThreadsPageTest.java

```diff
@@ -129,6 +129,24 @@ public void testHideAllThreads() {
 		Assert.assertFalse(chartCanvas.isContextMenuItemEnabled(RESET_CHART));
 	}
 
+	@Test
+	public void testPopupTableSelection() {
+		openThreadsTable();
+		final int numSelection = 7;
+		final int numThreads = threadsTable.getItemCount();
+		Assert.assertTrue(numThreads > 0 && numThreads >= numSelection);
+
+		threadsTable.selectItems(0, numSelection - 1);
+		int origonalSelection = threadsTable.getSelectionCount();
```

Comment:
s/origonal/original/ ?

Reply:
Fixed in https://github.com/aptmac/jmc/commit/b1a9f8de601f2b85e6ecdb93d4742059c4c9a5ad